Makefile

```diff
@@ -36,7 +36,7 @@ fmt:
 # Run all the linters
 lint: bin/golangci-lint bin/misspell
 	# TODO: fix disabled linter issues
-	./bin/golangci-lint run ./...
+	./bin/golangci-lint run ./... --max-issues-per-linter=0 --max-same-issues=0
```

Comment:
Can't we set these in the configuration file rather than using command line flags?

Reply:
Done!